<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SwitchWinNum</class>
 <widget class="QMainWindow" name="SwitchWinNum">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1298</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SwitchWinNum</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="id_frame_1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>461</width>
      <height>211</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="id_show_name_1">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>100</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>显示组名1</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="id_frame_2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>30</y>
      <width>461</width>
      <height>211</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="id_show_name_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>显示组名2</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="id_frame_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>250</y>
      <width>461</width>
      <height>211</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="id_show_name_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>显示组名3</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="id_frame_4">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>250</y>
      <width>461</width>
      <height>211</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="id_show_name_4">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>显示组名4</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="id_frame_5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>470</y>
      <width>461</width>
      <height>211</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="id_show_name_5">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>显示组名5</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="id_frame_6">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>470</y>
      <width>461</width>
      <height>211</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="id_show_name_6">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>显示组名6</string>
     </property>
    </widget>
   </widget>
   <widget class="GroupNumView" name="id_listView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>231</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="id_lab_group_list">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>组列表</string>
    </property>
   </widget>
   <widget class="QLabel" name="id_lab_widget_num">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>切换窗口个数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="id_btn_two_window">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>2个窗口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="id_btn_six_window">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>440</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>6个窗口</string>
    </property>
   </widget>
   <widget class="QLabel" name="id_lab_move">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>移动组号</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GroupNumView</class>
   <extends>QListView</extends>
   <header>groupnumview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
